lib/l10n/app_de.arb

```diff
@@ -275,5 +275,11 @@
   "cardPendingRideRejectDialogMessage": "Bist du sicher, dass du diese Mitfahrt ablehnen möchtest?",
   "cardPendingRideRejectDialogSuccessSnackBar": "Mitfahrt abgelehnt",
 
+  "pageChatEmptyTitle": "Nichts zu sehen",
+  "pageChatEmptyMessage": "Starte jetzt eine Konversation:)",
```

Comment:
there is a space missing here before the smiley

Reply:
thanks